README.md

```diff
@@ -573,3 +574,175 @@ export class MyAwesomeComponent extends React.PureComponent {
     }
 }
 ```
+## External libraries
+
+### Prefer native js vs lodash
+
+> Reason: lodash is a great library which contains a lot of useful functions but the modern js has the short identical part of this functionality so some lodash functions are excess alternative.
+
+> Don't forget to check necessary browsers compatibility and turn on polyfills if it's needed.
+```javascript
+// BAD
+_.each([1, 2, 3], callback); 
+// GOOD
+[1, 2, 3].forEach(callback); 
+```
+```javascript
+// BAD
+_.filter([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].filter(callback);
+```
+```javascript
+// BAD
+_.compact([1, 2, 3, 0, false, '']);
+// GOOD
+[1, 2, 3, 0, false, ''].filter(Boolean);
+```
+```javascript
+// BAD
+_.map([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].map(callback);
+```
+```javascript
+// BAD
+_.every([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].every(callback);
+```
+```javascript
+// BAD
+_.some([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].some(callback);
+```
+```javascript
+// BAD
+_.reduce([1, 2, 3], callback, 0);
+// GOOD
+[1, 2, 3].reduce(callback, 0);
+```
+```javascript
+// BAD
+_.concat([1, 2, 3], [4, 5, 6]);
+// GOOD
+[1, 2, 3].concat([4, 5, 6]);
+```
+```javascript
+// BAD
+_.indexOf([1, 2, 3], 1);
+// GOOD
+[1, 2, 3].indexOf(1);
+```
+```javascript
+// BAD
+_.includes([1, 2, 3], 1);
+// GOOD
+[1, 2, 3].includes(1);
+```
+```javascript
+// BAD
+const first = _.first([1, 2, 3]);
+// GOOD
+const [first] = [1, 2, 3];
+```
+```javascript
+// BAD
+_.join([1, 2, 3], '-');
+// GOOD
+[1, 2, 3].join('-');
+```
+```javascript
+// BAD
+_.reverse([1, 2, 3]);
+// GOOD
+[1, 2, 3].reverse();
+```
+```javascript
+// BAD
+_.uniq([1, 2, 3, 3, 2]);
+// GOOD
+[...new Set([1, 2, 3, 3, 2])];
```

Comment:
This one does not looks better vs lodash variant. `_.uniq` is more clear and shortener. Also, `Set` has cross browser limitations.

Reply:
Native and one line, easy to understand or to google purpose if not. I suggest it's a nice choice, IE should die, thinking about it is an atavism in 2k19. It will be attention about polyfills